Mathlib/Topology/Algebra/Module/LocallyConvex.lean

```diff
@@ -91,10 +91,21 @@ theorem locallyConvexSpace_iff_exists_convex_subset_zero :
   (locallyConvexSpace_iff_zero 𝕜 E).trans hasBasis_self
 
 -- see Note [lower instance priority]
-instance (priority := 100) LocallyConvexSpace.toLocallyConnectedSpace [Module ℝ E]
-    [ContinuousSMul ℝ E] [LocallyConvexSpace ℝ E] : LocallyConnectedSpace E :=
-  locallyConnectedSpace_of_connected_bases _ _
-    (fun x => @LocallyConvexSpace.convex_basis ℝ _ _ _ _ _ _ x) fun _ _ hs => hs.2.isPreconnected
+instance (priority := 100) LocallyConvexSpace.toLocPathConnectedSpace [Module ℝ E]
+    [ContinuousSMul ℝ E] [LocallyConvexSpace ℝ E] : LocPathConnectedSpace E :=
+  .of_bases (fun x ↦ @convex_basis ℝ _ _ _ _ _ _ x)
+    fun _ _ hs ↦ hs.2.isPathConnected <| nonempty_of_mem <| mem_of_mem_nhds hs.1
+
+/-- Convex subsets of locally convex spaces are locally path-connected. -/
+theorem Convex.locPathConnectedSpace [Module ℝ E] [ContinuousSMul ℝ E] [LocallyConvexSpace ℝ E]
+    {S : Set E} (hS : Convex ℝ S) : LocPathConnectedSpace S := by
+  refine ⟨fun x ↦ ⟨fun s ↦ ⟨fun hs ↦ ?_, fun ⟨t, ht⟩ ↦ mem_of_superset ht.1.1 ht.2⟩⟩⟩
+  let ⟨t, ht⟩ := (mem_nhds_subtype S x s).mp hs
+  let ⟨t', ht'⟩ := (@LocallyConvexSpace.convex_basis ℝ _ _ _ _ _ _ x.1).mem_iff.mp ht.1
```

Comment:
Can you use named arguments instead of a sea of underscores? Something like `LocallyConvexSpace.convex_basis (\bbk := ℝ) x.1` (untested)